### DISCUSSION TITLE
Offer option to publish secure spreadsheet as table on listed.to

### DISCUSSION THREAD

**Discussion by archerships** [standardnotes/forum#2633] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
Secure spreadsheets published to listed.to blog look like gobbleygook.  For example: 

https://listed.dja.la/22276/

**Describe the solution you'd like**
I would like to: 

a.  publish spreadsheets on listed.to as html tables 
b.  offer download links, for CSV, Markdown Table text files, and .XLS files  

**Describe alternatives you've considered**
* Manually converting spreadsheet in other software and hosting elsewhere.  
* Not using secure spreadsheet, and writing out table in Markdown by hand.

**Additional context**
![Screenshot from 2021-01-28 16-44-51](https://user-images.githubusercontent.com/77993343/106216562-440bc280-6188-11eb-9fd9-d4b95f57bbf5.png)

**Comment by JaspalSuri** [comment]
Hi @archerships,

Thanks for the suggestion! While I don't think we'll implement this feature in the near future, you can copy the cells from the Secure Spreadsheets editor and paste them into a new note using either the Plus or Bold editor, before publishing the note to your blog (or a private post).

**Comment by archerships** [comment]
Thanks for the suggestion!

**Comment by JaspalSuri** [comment]
You're welcome! 🙂
